blog/_posts/2016-11-08-RubyConfPt.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Attending the Ruby Conference in Portugal
+layout: post
+created_at: Tues Nov 08 2016
+permalink: blog/2016-11-08-RubyConfPt
+current: blog
+author: Thea Amanda Kupler, Marie-Luise Kochsiek, Mayar Alaa and Nada Ashraf
+twitter: loadtocode
```

Comment:
Unfortunately, adding more than one twitter account doesn't work :( adding the account handles like this will concatenate the two handles into one. Can you settle for one account/handle to link only? If you want you can add an extra line at the bottom of the post where you link to both your twitter accounts again (or to your personal ones, whichever way you like).